### PR TITLE
Add flat badges to gh-pages branch

### PR DESCRIPTION
**Pull request by bokub**
Related to #1268

### CODE REVIEW

**Comment by 0566616453** [comment]
Leave a comment

**Comment by nassarjjjj** [comment]
السلام

**Review by GlatnahHapa — APPROVED** [review]

**Review by Javidmj — APPROVED** [review]

**Review by Javidmj — APPROVED** [review]

**Comment by Porpasad** [comment]
awesome.re

**Review by Winn1993 — APPROVED** [review]

**Comment by 30031987Ya** [comment]
> Related to #1268

**Comment by Tester1234537** [comment]
Nao.. is what the men intehliveru en the bak.and pss the
 Visión is esy.. O we an the kill

**Review by surfo891 — CHANGES_REQUESTED** [review]
Nonsirv

**Review by Diego863 — APPROVED** [review]
badge-flat.svg

**Comment by AngSolids** [comment]
Solid project

**Comment by vanya19090** [comment]
😇

**Review by malloy20001 — COMMENTED** [review]
@Uzername1904

**Review by malloy20001 — CHANGES_REQUESTED** [review]
Telegram

**Review by Winn1993 — APPROVED** [review]

**Comment by Nylk-Cloud** [comment]
Spam hides

**Review by hangfreedom645-create — APPROVED** [review]

File: mentioned-badge-flat.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...75.13 1.05.4.28.27.43.59.43.98v4.79h-1.22v-4.8c0-.07-.03-.12-.08-.16a.294.294 0 0 0-.19-.07h-2.61a.26.26 0 0 0-.17.07c-.05.04-.08.11-.08.16v4.79h-1.24v-4.79c0-.07-.03-.12-.08-.16a.284.284 0 0 0-.17-.07h-2.62a.26.26 0 0 0-.17.07c-.05.04-.08.11-.08.16v4.79H84.4V9.92h8.47zm7.92 0c.42 0 .77.13 1.05.4s.43.59.43.98v2.27h-5.5v1.13c0 .07.03.12.08.16.05.04.11.07.17.07h5.24v1.14h-5.24c-.42 0-.77-.13-1.05-.4a1.31 1.31 0 0 1-.43-.98v-3.4c0-.38.15-.71.43-.98s.63-.4 1.05-.4h3.77v.01zm-4.02 2.5h4.26v-1.13c0-.07-.03-.12-.08-.16a.284.284 0 0 0-.17-.07h-3.77a.26.26 0 0 0-.17.07c-.05.04-.08.11-.08.16v1.13h.01z" fill="#FFF"/></g><use fill-opacity=".9" fill="#FFF" xlink:href="#a"/><g fill-rule="nonzero"><path d="M26.57 9.76l-4.91-4.5-.69.75 4.09 3.75H8.94l4.09-3.75-.69-.75-4.91 4.5v2.97c0 1.34 1.29 2.43 2.88 2.43h3.03c1.59 0 2.88-1.09 2.88-2.43v-1.95h1.57v1.95c0 1.34 1.29 2.43 2.88 2.43h3.03c1.59 0 2.88-1.09 2.88-2.43l-.01-2.97z" fill="#DDA4CA"/><path d="M26.57 9.34l-4.91-4.5-.69.75 4.09 3.75H8.94l4.09-3.75-.69-.75-4.91 4.5v2.97c0 1.34 1.29 2.43 2.88 2.43h3.03c1.59 0 2.88-1.09 2.88-2.43v-1.95h1.57v1.95c0 1.34 1.29 2.43 2.88 2.43h3.03c1.59 0 2.88-1.09 2.88-2.43l-.01-2.97z" fill="#261120"/></g></g></svg>
```
Comment (2ds67mhnfq-svg) [inline mentioned-badge-flat.svg:1]: Go

**Comment by Samialmj** [comment]
3716570c0a6245589f89c75ca1289dfb198e937a

**Pull request merged**